### ISSUE TITLE
[BUG] [macOS] Eww window is only visible on one space

### ISSUE THREAD

**Issue by Rougemoot** [elkowar/eww#65]
## Describe the bug
When on macOS, the eww window is only visible on the one space it's  been started from.

## Reproducing the issue
Use eww on macOS.

## Expected behaviour
The window should be sticky, appearing on all spaces or following the user around

## Additional context
I guess that could either be fixed by actually making the window visible on all spaces, or it could also be solved by making the window a proper detectable window according to macOS, which would emable the user to make it sticky through their WM

**Comment by elkowar** [comment]
@Rougemoot can you try setting `focusable="true"` on your window definition?
as in:
```xml
<window name="main_window" selectable="true">
  ...
</window>
```

That would tell the WM to treat eww more like a normal window, specifically also making it _focusable_. This may make it possible for you to go with your option 2, making it sticky in the WM config. Note that that is likely to also make eww focusable by the WM, which is something you'd then also need to tell the WM to ignore in some other way

**Comment by Rougemoot** [comment]
I'll definitely try that, although is is `focusable` or `selectable`?

**Comment by elkowar** [comment]
focusable, sorry xD Just noticed my mistake

**Comment by elkowar** [comment]
Added the `focusable` attribute to the docs, among a couple others in 0e97847a3ae4614cc1dcb7cf51ece2e63745c97f

**Comment by Rougemoot** [comment]
Unfortunately this doesn't solve the problem. The window still isn't detected by my WM (yabai) so I can't make it sticky.
Two things have changed from adding the `selectable` property: a border appeared around the window, and all text becomes grey unless I click on the window, as if to indicate that the window isn't currently focused.

**Comment by elkowar** [comment]
i guess the border and focus stuff was to be expected, lol. It's kinda weird that it isn't detected by the WM tho,... do you have any way of seeing what makes the WM ignore it?

**Comment by Rougemoot** [comment]
Nope, I can just ask the WM what it sees, and it just doesn't see it as a proper window

**Comment by elkowar** [comment]
not as a _proper_ window (if so, what exactly does that mean? what _does_ it see it as?) or just not at all?

**Comment by Rougemoot** [comment]
Oh yeah sorry it just doesn't see it at all.

**Comment by Rougemoot** [comment]
This might be what we need?
https://wiki.gnome.org/Projects/GTK/OSX

**Comment by Axarva** [comment]
@Rougemoot Do you mean the widgets...
1. ...start in a place, are visible there, and then if you launch a window it gets covered up?
2. ...always just keeps getting started behind windows?

Because if it's the first one, that's what the default behavior is on my window manager as well.

**Comment by elkowar** [comment]
1. may be a result of xmonad not respecting the window stacking hints, that may or may not be fixed in the gtk4-version, or may require some xmonad shit that i'll need to investigate. 

Vermoots issue here is workspaces, eww on macos is currently not "sticky", and cannot be made such as it isn't seen by yabai at all. That, too, may be fixed by gtk4.
